src/locale/en/attributeSelectionEn.ts

```diff
@@ -9,4 +9,6 @@ export default {
     $phytoplankton: 'Phytoplankton',
     $benthicFauna: 'Benthic fauna',
     $selectAll: 'Select all',
+    $phosphorusMessage: 'Anomalies have been observed in phosphorus data of open sea stations from January 1, 2000 to February 28, 2014. The data is under checking and can’t be loaded for the time being.',
```

Comment:
Oisko "data is under checking" paremmin sanottuna "the data is being verified"?

Reply:
Mä sain tän käännöksen Heikiltä... En mä tiedä, kai se on ok noinkin

Reply:
Joo okei, jos tuli tosiaan ylempää niin antaa olla